dali/operators/image/distortion/jpeg_compression_distortion_op_cpu.cc

```diff
@@ -85,10 +85,15 @@ void JpegCompressionDistortionCPU::RunImpl(workspace_t<CPUBackend> &ws) {
     auto shape = in_shape.tensor_shape_span(sample_idx);
     int ndim = shape.size();
     auto layout = input.GetLayout();
+
     int w_dim = layout.find('W');
+    assert(w_dim >= 0);
     int h_dim = layout.find('H');
+    assert(h_dim >= 0);
     int c_dim = layout.find('C');
+    assert(c_dim >= 0);
     int f_dim = layout.find('F');
+
```

Comment:
If there are asserts for every value here, why no assert for `f_dim`?

Reply:
because we allow it to be -1

Reply:
See lines 97-98